templates/terraform/post_create/cloudbuild_trigger_id.go.erb

```diff
@@ -0,0 +1,14 @@
+// `name` is autogenerated from the api so needs to be set post-create
```

Comment:
We have `set_computed_name.erb` for the common `name` case (similar to update mask) but this one is different since we're recording the `id`, and this is one of the only non-numeric `id` values. I don't think this is worth templating yet since this particular pattern is still isolated. If we get to 3 that are nearly identical imo that's when it's worth it.